iosApp/Survey/Sources/Presentation/Modules/Login/LoginView.swift

```diff
@@ -77,18 +77,18 @@ struct LoginView: View {
     }
 
     var loginField: some View {
-        TextField(Localize.loginFieldEmail(), text: $email)
+        TextField(LocalizeId().login_fields_email.localized, text: $email)
```

Comment:
Just a thought, but maybe we can extract `LocalizeId()` to a constant for reusing. Then we can clean up the Typealias declaration for `MR.strings` for simplicity. What do you think? 🤔

Reply:
Updated to a singleton in https://github.com/blyscuit/kmm-swiftUI-IC/pull/76/commits/8d925c48c3c6d8ae39e13d428293a1ce42ac6e63